examples/reference/widgets/NestedSelect.ipynb

```diff
@@ -24,6 +24,7 @@
     "\n",
     "* **``options``** (dict | callable): The options to select from. The options may be nested dictionaries, lists, or callables that return those types. If callables are used, the callables must accept `level` and `value` keyword arguments, where `level` is the level that updated and `value` is a dictionary of the current values, containing keys up to the level that was updated.\n",
     "* **``value``** (dict): The value from all the Select widgets; the keys are the levels names. If no levels names are specified, the keys are the levels indices.\n",
+    "* **``layout``** (ListLike | dict): The layout type of the widgets. If a dictionary, a \"type\" key can be provided, to specify the layout type of the widgets, and any additional keyword arguments will be used to instantiate the layout.\n",
```

Comment:
@ahuang11 do you know if we expose `ListLike` as a public API?

Reply:
Yeah ListLike is incorrect, ListPanel seems like the correct type.

Reply:
I chose ListLike because it covers more layouts, like FlexBox:

`class FlexBox(ListLike, ReactiveHTML):`

Reply:
In that case we need another baseclass, `ListLike` just implements a list-like API it is not a Panel component type.

Reply:
Searching the codebase, I suppose FlexBox is the only one with ListLike.